### ISSUE TITLE
How to define the hidden size of LSTM

### ISSUE THREAD

**Issue by junlulocky** [fchollet/keras#4743]
in lstm_benchmark.py, I find to define a LSTM: LSTM(embedding_dim, dropout_W=0.2, dropout_U=0.2, consume_less=mode)

But how can I define the hidden output size of if, i.e. how can I define the second dimension of the weights? I do look all the examples and some of the source codes, but I did not find out. Can you explain a little bit for me?

**Comment by jaredsagendorf** [comment]
Assuming Keras follows the implementation described Hochreiter & Schmidhuber, the hidden state is the same size as the cell state, or output size.

**Comment by meysamgolm** [comment]
Look at below. lstm_layer_size_a is the hidden output size.

`

    # build LSTM and train it.
    #
    model = Sequential()
    model.add(LSTM(lstm_layer_size_a, input_shape=train_set_x.shape[1:]))
    model.add(Dense(1, activation='sigmoid'))
    model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])
    print(model.summary())
    model.fit(train_set_x, train_set_y, validation_data=(train_set_x, train_set_y),
          nb_epoch=lstm_epoch_size_a, batch_size=lstm_batch_size_a)`

**Comment by lujunzju** [comment]
Hi, I think the lstm_layer_size_a here is the output size, which is not the
hidden size, I think jaredsagendorf might be right, the hidden size is the
same with output size in Keras.
